Pedigree.py

```diff
@@ -244,6 +246,10 @@ def __init__(self, fileName = None, constructor = Individual):
 
         self.referencePanel = [] #This should be an array of haplotypes. Or a dictionary?
 
+        self.MainMetaFounder = None
+        self.AAP = {}
+
+        # remove?
```

Comment:
@XingerTang about this comment (`remove?`) are you thinking of removing `self.maf`? If so, where would we be storing maf?

Reply:
@gregorgorjanc In alphapeel the maf are stored in the peelingInfo, and this self.maf is not used. But I don't know about other software so I put a question mark.

Reply:
OK, future work.